config/schemas/settings.json

```diff
@@ -48,6 +55,34 @@
 					"type" : "string",
 					"default" : "Player"
 				},
+				"playerName2" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName3" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName4" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName5" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName6" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName7" : {
+					"type" : "string",
+					"default" : ""
+				},
+				"playerName8" : {
+					"type" : "string",
+					"default" : ""
+				},
```

Comment:
Why not use array instead?

Reply:
Could be an array for all the names including the first player's one? The drawback would be that configs would become incompatible with older versions.

Reply:
Option a - use "hotseatNames" that would include only names for non-first players. Probably easiest one.
Option b - import old data in some way. For example, add property with different name, e.g. "playerNames" and if first name empty - set it to current value of old "playerName" property

Reply:
Yup I was thinking about the second one. I'll play with it, thanks!